src/store/module-map/actions.js

```diff
@@ -46,6 +48,17 @@ function _makeBaseLayer (baseLayer) {
       layer = L.tileLayer(baseLayer.url, baseLayer)
     } else if (baseLayer.type === 'wms') {
       layer = L.tileLayer.wms(baseLayer.url, baseLayer)
+    } else if (baseLayer.type === 'googleMutant') {
```

Comment:
This layer type should be optional, please do some check and warning like in `geomUtils:191`